Alicerce/Sources/Extensions/Collection.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Collection.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 18/12/2016.
+//  Copyright © 2016 Mindera. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - Array
+
+func -<Key: Hashable, Value: AnyObject>(left: [[Key : Value]], right: [[Key : Value]]) -> [[Key : Value]] {
```

Comment:
It would be nice to add some documentation to these operators 😇 

Personally, I prefer documenting with three slashes `///` instead of `/**/`.